com.unity.render-pipelines.core/Runtime/Textures/PowerOfTwoTextureAtlas.cs

```diff
@@ -233,42 +233,40 @@ public override bool AllocateTexture(CommandBuffer cmd, ref Vector4 scaleOffset,
         public void ResetRequestedTexture() => m_RequestedTextures.Clear();
 
         /// <summary>
-        /// Reserve space from atlas for a texture.
+        /// Reserves the space on the texture atlas
         /// </summary>
-        /// <param name="texture">Source texture.</param>
-        /// <returns>True on success, false otherwise.</returns>
+        /// <param name="texture">The source texture</param>
+        /// <returns>True if the space is reserved</returns>
         public bool ReserveSpace(Texture texture) => ReserveSpace(texture, texture.width, texture.height);
 
         /// <summary>
-        /// Reserve space from atlas for a texture.
+        /// Reserves the space on the texture atlas
         /// </summary>
-        /// <param name="texture">Source texture.</param>
-        /// <param name="width">Request width in pixels.</param>
-        /// <param name="height">Request height in pixels.</param>
-        /// <returns>True on success, false otherwise.</returns>
+        /// <param name="texture">The source texture</param>
+        /// <param name="width">The width</param>
+        /// <param name="height">The height</param>
+        /// <returns>True if the space is reserved</returns>
         public bool ReserveSpace(Texture texture, int width, int height)
             => ReserveSpace(GetTextureID(texture), width, height);
 
-
         /// <summary>
-        /// Reserve space from atlas for a texture.
-        /// Pass width and height for CubeMap (use 2*width) & Texture2D (use width).
+        /// Reserves the space on the texture atlas
```

Comment:
IMO should be "Reserves space from the texture atlas".

Also seems like the removed extra info could be useful to someone so why not keep it?

Reply:
```suggestion
        /// Reserves space from the texture atlas
```